src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1945,11 +1950,12 @@
     <Field Name="Property" Type="PropertySymbol" Null="disallow"/>
   </Node>
   
-  <Node Name="BoundNewT" Base="BoundExpression">
+  <Node Name="BoundNewT" Base="BoundObjectCreationExpressionBase">
```

Comment:
Can we remove `<Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>` from `BoundNewT` and `BoundNoPiaObjectCreationExpression` now?

Reply:
> Can we remove ```<Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>``` from ```BoundNewT``` and ```BoundNoPiaObjectCreationExpression``` now?

Yes, I think those can be removed.